dali/operators/math/expressions/expression_impl_gpu.cuh

```diff
@@ -33,8 +33,12 @@ namespace dali {
 template <ArithmeticOp op, typename Result, typename Input>
 __device__ void ExecuteUnOp(Result *result, const Input *in, int64_t extent) {
   using meta = arithm_meta<op, GPUBackend>;
+  result += blockIdx.x * blockDim.x + threadIdx.x;
+  in += blockIdx.x * blockDim.x + threadIdx.x;
   for (int64_t i = blockIdx.x * blockDim.x + threadIdx.x; i < extent; i += blockDim.x * gridDim.x) {
-    result[i] = meta::impl(in[i]);
```

Comment:
actually it seemed more readable before. Why the change?

Reply:
For a small perf gain as well as correctness with int64 offset.